README.md

```diff
@@ -0,0 +1,16 @@
+# `libftdi1-sys`
+`libftdi1-sys` is a crate providing Rust bindings to the C library
+[`libftdi1`](https://www.intra2net.com/en/developer/libftdi/index.php).
+
+## Prerequisites
+The libftdi1 bindings are generated at compile-time, per bindgen's
+[recommended usage](https://rust-lang.github.io/rust-bindgen/library-usage.html).
+
+* `libclang` must be installed and visible on your path. If you are using a
+  `gcc`-toolchain and don't want to install the entirity of LLVM just for
+  `libclang`,
+
+* `libftdi` version 1.4 (the most recent version as of 2017) is required.
+
+* The Minimum Supported Rust Version (MSRV) is stable `1.42`. However, older
```

Comment:
This is not strictly correct. You don't use the bindgen `rust_target` option, so the target by default is "the latest stable Rust version". However, as bindgen seems to almost never do compatible version bumps, this is likely to be true.

Still, this requirement sounds a bit suboptimal. I'll discuss it in a bit more detail in a follow-up comment.